### PR TITLE
[13.0][MIG] account_payment_return_import: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT19692

### CODE REVIEW

File: account_payment_return_import/tests/test_import_base.py [hunk 1]
```diff
@@ -27,62 +27,9 @@ def setUpClass(cls):
                 "name": "Test Bank Journal",
                 "code": "BANK",
                 "type": "bank",
-                "update_posted": True,
                 "bank_account_id": cls.acc_bank.id,
             }
         )
-        cls.journal.bank_account_id = cls.acc_bank
```
Comment (pedrobaeza) [inline account_payment_return_import/tests/test_import_base.py]: Why removing all this stuff? This is probably needed for subsequent modules.

File: account_payment_return_import/wizard/payment_return_import.py [hunk 2]
```diff
@@ -19,11 +19,11 @@ class PaymentReturnImport(models.TransientModel):
     _name = "payment.return.import"
     _description = "Import Payment Return"
 
-    @api.model
     def _compute_hide_journal_field(self):
         """ Return False if the journal_id can't be provided by the parsed
         file and must be provided by the wizard."""
-        return True
+        for record in self:
+            record.hide_journal_field = True
```
Comment (pedrobaeza) [inline account_payment_return_import/wizard/payment_return_import.py]: In v13 an assignation over a whole recordset works:
```suggestion
        self.hide_journal_field = True
```

**Review by pedrobaeza — APPROVED** [review]

File: account_payment_return_import/tests/test_import_base.py (hunk 1 above)
Comment (ernestotejeda) [reply account_payment_return_import/tests/test_import_base.py]: I hesitated to delete it, and yet I deleted it. Next time I have doubts about deleting something or not in a base module I wont delete anything, haha.

File: account_payment_return_import/wizard/payment_return_import.py (hunk 2 above)
Comment (ernestotejeda) [reply account_payment_return_import/wizard/payment_return_import.py]: I didn't know, thanks. It's already changed

**Comment by ernestotejeda** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]
Code review

**Comment by pedrobaeza** [comment]
Please check Travis

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**